adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapter.java

```diff
@@ -286,64 +286,60 @@ private void handleSessionOpen(final ProtonConnection conn, final ProtonSession
     }
 
     /**
-     * This method is invoked when an AMQP Attach frame is received by this server.
+     * This method is invoked when an AMQP Attach frame is received by this server. If the receiver link contains
+     * a target address, this method simply closes the link, otherwise, it accept and open the link.
      * 
      * @param receiver The receiver link for receiving the data.
      * @param conn The connection through which the request is initiated.
      */
     protected void handleRemoteReceiverOpen(final ProtonReceiver receiver, final ProtonConnection conn) {
-        final Device authenticatedDevice = conn.attachments().get(AmqpAdapterConstants.KEY_CLIENT_DEVICE,
-                Device.class);
-        validateEndpoint(receiver, authenticatedDevice).recover(t -> {
-            LOG.debug(
-                    "fail to establish a receiving link with client [{}] due to an invalid endpoint [{}]."
-                            + " closing the link",
-                    receiver.getName(), receiver.getRemoteQoS());
-            receiver.setCondition(AmqpContext.getErrorCondition(t));
+        if (receiver.getRemoteTarget() != null) {
+            if (receiver.getRemoteTarget().getAddress() != null && !receiver.getRemoteTarget().getAddress().isEmpty()) {
+                LOG.debug("Closing link due to the present of Target [address : {}]", receiver.getRemoteTarget().getAddress());
+            }
+            receiver.setCondition(
```

Comment:
why don't you allow a Target without an *address*?

Reply:
sorry, i was sleeping here..

Reply:
now i understand what you mean :+1: